en/sim_gazebo_gz/README.md

```diff
@@ -105,6 +105,10 @@ However you must not _explicitly_ specify `_default` on the model as this will p
 In other words, use `make px4_sitl gz_x500` instead of `make px4_sitl gz_x500_default` for the default.
 :::
 
+:::warning
```

Comment:
I think a note. It's important, but not "scary"

```suggestion
:::note
```

Reply:
@hamishwillee I don't know. If I see a bright red error message, I would read it and ctrl + C out of Gazebo. I think it's important to highlight that this error is expected.